### PR TITLE
Fix connection pool closing issue in test cases

### PR DESCRIPTION
**Pull request by anupama-pathirage**

### CODE REVIEW

File: modules/ballerina-native/src/test/resources/samples/datatableTest.bal [hunk 1]
```diff
@@ -82,11 +82,18 @@ function testToJson () (json) {
...
-    json result;
-    result, _ = <json>dt;
-    return result;
+    try {
+        datatable dt = testDB.select("SELECT int_type, long_type, float_type, double_type,
+                  boolean_type, string_type from DataTable WHERE row_id = 1", parameters);
+        json result;
+        result, _ = <json>dt;
+        return result;
+    } catch (errors:Error err) {
```
Comment (hasithaa) [inline modules/ballerina-native/src/test/resources/samples/datatableTest.bal]: we can remove catch block.

File: modules/ballerina-native/src/test/resources/samples/datatableTest.bal [hunk 2]
```diff
@@ -95,11 +102,18 @@ function testToXml () (xml) {
...
-    xml result;
-    result, _ = <xml>dt;
-    return result;
+    try {
+        datatable dt = testDB.select("SELECT int_type, long_type, float_type, double_type,
+                   boolean_type, string_type from DataTable WHERE row_id = 1", parameters);
+        xml result;
+        result, _ = <xml>dt;
+        return result;
+    } catch (errors:Error err) {
```
Comment (hasithaa) [inline modules/ballerina-native/src/test/resources/samples/datatableTest.bal]: we can remove catch block.

File: modules/ballerina-native/src/test/resources/samples/datatableTest.bal [hunk 3]
```diff
@@ -108,12 +122,19 @@ function toXmlComplex () (xml) {
...
-    result, _ = <xml>dt;
-    return result;
+    try {
+        datatable dt = testDB.select("SELECT int_type, int_array, long_type, long_array, float_type,
+                    float_array, double_type, boolean_type, string_type, double_array, boolean_array, string_array
+                    from MixTypes where row_id =1", parameters);
+        xml result;
+        result, _ = <xml>dt;
+        return result;
+    } catch (errors:Error err) {
```
Comment (hasithaa) [inline modules/ballerina-native/src/test/resources/samples/datatableTest.bal]: remove catch block.

File: modules/ballerina-native/src/test/resources/samples/datatableTest.bal [hunk 4]
```diff
@@ -197,11 +218,18 @@ function testJsonWithNull () (json) {
...
-    json result;
-    result, _ = <json>dt;
-    return result;
+    try {
+        datatable dt = testDB.select("SELECT int_type, long_type, float_type, double_type,
+                  boolean_type, string_type from DataTable WHERE row_id = 2", parameters);
+        json result;
+        result, _ = <json>dt;
+        return result;
+    }  catch (errors:Error err) {
```
Comment (hasithaa) [inline modules/ballerina-native/src/test/resources/samples/datatableTest.bal]: remove catch block.

File: modules/ballerina-native/src/test/resources/samples/datatableTest.bal [hunk 5]
```diff
@@ -210,11 +238,18 @@ function testXmlWithNull () (xml) {
...
-    xml result;
-    result, _ = <xml>dt;
-    return result;
+    try {
+        datatable dt = testDB.select("SELECT int_type, long_type, float_type, double_type,
+                   boolean_type, string_type from DataTable WHERE row_id = 2", parameters);
+        xml result;
+        result, _ = <xml>dt;
+        return result;
+    } catch (errors:Error err) {
```
Comment (hasithaa) [inline modules/ballerina-native/src/test/resources/samples/datatableTest.bal]: remove catch block.

File: modules/ballerina-native/src/test/resources/samples/datatableTest.bal (hunk 1 above)
Comment (anupama-pathirage) [inline modules/ballerina-native/src/test/resources/samples/datatableTest.bal]: Fixed with 5d0d8264ab9473e0749edf6cc43576cf119d699d . Thanks

**Pull request merged**